web/app/scripts/views/record/add-edit-controller.js

```diff
@@ -18,10 +20,14 @@
             ctl.onSaveClicked = onSaveClicked;
             ctl.occurredFromChanged = occurredFromChanged;
             ctl.onGeomChanged = onGeomChanged;
+            ctl.geocode = geocode;
+            ctl.geocodeSelect = geocodeSelect;
 
             ctl.occurredFromOptions = {format: dateTimeFormat};
             ctl.occurredToOptions = {format: dateTimeFormat};
 
+            ctl.reverseGeocoder = '';
```

Comment:
This name can be a bit more descriptive for what it represents. Probably match it up for whatever is chosen for the label of that field.